client/offline.go

```diff
@@ -0,0 +1,111 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package client
+
+import (
+	"crypto"
+	"crypto/sha256"
+
+	"github.com/google/trillian"
+	tcrypto "github.com/google/trillian/crypto"
+	"github.com/google/trillian/merkle"
+)
+
+// LogVerifier contains state needed to verify output from Trillian Logs.
+type LogVerifier struct {
```

Comment:
As it stands, I don't see much benefit in having LogVerifier. It's basically duplicating the merkle.LogVerifier API, with little added benefits (other than a few conversions).

If you do intent to keep it, please add tests for all public methods.